src/OpenIddict.Validation.DataProtection/OpenIddictValidationDataProtectionFormatter.cs

```diff
@@ -35,21 +35,10 @@ public ClaimsPrincipal ReadToken(BinaryReader reader)
 
             .SetClaim(Claims.Private.HostProperties, GetJsonProperty(properties, Properties.HostProperties))
 
-            .SetClaim(Claims.Private.AccessTokenLifetime,       GetProperty(properties, Properties.AccessTokenLifetime))
```

Comment:
Note: most of these mappings were useless as the validation stack only processes DP access tokens and these properties/claims are not present in access tokens.